docs/guides/how_to_guides/validation/how_to_create_a_new_checkpoint.rst

```diff
@@ -66,7 +66,7 @@ Steps
 
             config = """
             name: my_checkpoint
-            config_version: 1
+            config_version: 1.0
```

Comment:
I am not sure if this should change to float.  Hope we can check and make consistent.